receiver/apachereceiver/config.go

```diff
@@ -28,6 +28,7 @@ type Config struct {
 	scraperhelper.ScraperControllerSettings `mapstructure:",squash"`
 	confighttp.HTTPClientSettings           `mapstructure:",squash"`
 	serverName                              string
+	emitServerNameAsResourceAttribute       bool
```

Comment:
This is still a config option, not a feature gate. Please take a look at this feature gates example https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/6457705e332a8d531e82ffea38c0f6d3ba1337d7/receiver/zookeeperreceiver/scraper.go#L45

Reply:
There is a feature gate used, just not directly in the scraper, but in the factory:
https://github.com/aboguszewski-sumo/opentelemetry-collector-contrib/blob/6bac04c5014dbda6dfe8a424c02b9ab93459cbcc/receiver/apachereceiver/factory.go#L38-L50
I just thought that a pattern where it is injected in the scraper with the config seems more reasonable. I'll just move this code to the scraper then.

Reply:
Changed.